### DISCUSSION TITLE
How to use cloudsimplus for HPA (Auto-Scaling) experiment?

### DISCUSSION THREAD

**Discussion by Li-ChenHao** [manoelcampos/cloudsim-plus#423] [Q&A] [answered]
Does Cloudsimplus support the HPA(Horizontal Pod Autoscaler) mechanism(like Kubernetes)? I hope to use Cloudsimplus to perform HPA improvement algorithms, and I don't know if it is feasible.

**Comment by manoelcampos** [comment, marked as answer]
Hello @Li-ChenHao 
I don't know this HPA mechanism, but check Vertical (#7) and Horizontal VM scaling (#41) features.
